src/component/mxgraph/StyleUtils.ts

```diff
@@ -23,15 +23,15 @@ export enum MarkerIdentifier {
 export enum StyleDefault {
   STROKE_WIDTH_THIN = 2,
   STROKE_WIDTH_THICK = 5,
-  TASK_SHAPE_BOTTOM_MARGIN = 7,
+  SHAPE_ACTIVITY_BOTTOM_MARGIN = 7,
   DEFAULT_FILL_COLOR = 'White',
   DEFAULT_STROKE_COLOR = 'Black',
   DEFAULT_FONT_FAMILY = 'Arial, Helvetica, sans-serif', // define our own to not depend on eventual mxGraph default change
   DEFAULT_FONT_SIZE = 11,
   DEFAULT_FONT_COLOR = 'Black',
   DEFAULT_MARGIN = 0,
-  DEFAULT_DASHED = 0,
-  DEFAULT_FIX_DASH = 0,
+  DEFAULT_DASHED = 0, // it means 'false'
```

Comment:
```suggestion
  DEFAULT_DASHED = false,
```

Reply:
Forget. I didn't see your PR's description ^^

Reply:
I tried this, but I got: `TS2553: Computed values are not permitted in an enum with string valued members.`

Reply:
I had a similar error before. It was because there is a error in the typescript's definition of MxGraph.

Reply:
the error is not related to mxgraph types: I got the error in our enum, not when calling mxgraph code.
The issue you are talking about is fixed in mxgraph-type-definitions (mxgraph canvas dashPattern and setFixDash - I am not fully sure of the exact name)